packages/openid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@user-office-software/openid",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Maybe you forgot to run `npm run lerna:prePublish` to update the package version in the package-lock.json as well